apps/server/src/shared/domain/entity/user.entity.ts

```diff
@@ -13,6 +14,28 @@ export enum LanguageType {
 	UK = 'uk',
 }
 
+export interface Consent {
+	userConsent?: UserConsent;
+	parentConsents?: ParentConsent[];
+}
+
+export interface UserConsent {
+	form: string;
```

Comment:
The ParentConsent has an _id and the UserContent not. Is this correct?

Reply:
yes